src/Game.js

```diff
@@ -7,15 +7,66 @@ function Game() {
 
 Game.prototype.roll = function (score) {
   for (var i = 1; i <= 11; i++) {
-    if (this.rollHistory[i].length < 2 && this.rollHistory[i][0] != 10 && i != 11) {
+    if (this._isNormalFrame(i)) {
```

Comment:
You have shown a clear approach to refactoring :) This is now much easier to read than before :)